__tests__/views/signin.view.test.js

```diff
@@ -1,10 +1,23 @@
+// import React from 'react';
```

Comment:
Good Job @Jajabenit250  but can you delete all these commented files if you are not using them.